examples/universal-with-router/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "redux-universal-example",
+  "version": "0.0.0",
+  "description": "An example of a universally-rendered Redux application",
+  "scripts": {
+    "start": "node server/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/reactjs/redux.git"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/reactjs/redux/issues"
+  },
+  "homepage": "http://redux.js.org",
+  "dependencies": {
+    "babel-polyfill": "^6.3.14",
+    "babel-register": "^6.4.3",
+    "express": "^4.13.3",
+    "qs": "^4.0.0",
+    "react": "^0.14.7",
+    "react-dom": "^0.14.7",
+    "react-redux": "^4.2.1",
+    "react-router": "^2.0.0",
+    "react-router-redux": "^3.0.0",
```

Comment:
Can we please use 4.x instead? I think there’s a beta out. It has a slightly different API but it’s better.